py-polars/polars/sql/context.py

```diff
@@ -203,7 +203,6 @@ def __init__(
         if register_globals:
             for name, obj in _get_frame_locals(
                 all_compatible=False,
-                n_objects=None if (register_globals is True) else None,
```

Comment:
😄 this made me smile

looks like `n = None if register_globals is True else None` was introduced in https://github.com/pola-rs/polars/pull/8749/files and then refactored in https://github.com/pola-rs/polars/pull/8762/files - @alexander-beedie do you happen to remember the intention behind it?

Reply:
Errr... possibly being a muppet, heh. Best guess is I was initially using it for local debugging with some other (non-None) value, and then forgot to set it back to _just_ being `None` 🤣